### DISCUSSION TITLE
Verbose debugpy adapter startup

### DISCUSSION THREAD

**Discussion by alessiocelentano** [mfussenegger/nvim-dap#296] [Q&A] [answered]
Basically, when i debug my Python code, debugpy prints this string: "\nptvsd\ndebugpy\n", I'd like to remove these lines, is there a way?

My configuration:
```
dap.adapters.python = {
  type = 'executable';
  command = '/usr/bin/python3';
  args = { '-m', 'debugpy.adapter' };
}

dap.configurations.python = {
  {
    type = 'python';
    request = 'launch';
    name = "Launch file";
    program = "${file}";
  },
}
```

Note: these lines aren't printed if I run the code with debugpy on my shell

**Comment by mfussenegger** [comment, marked as answer]
Looks like these are output events of the telemetry category sent by the debug adapter. https://github.com/mfussenegger/nvim-dap/pull/305 changes the behavior to log them instead.
